app/assets/stylesheets/application.css.scss

```diff
@@ -15,4 +15,35 @@
  */
 
 @import 'bootstrap-sprockets';
-@import 'bootstrap';
\ No newline at end of file
+@import 'bootstrap';
+.head {
```

Comment:
we have custom.css.scss for adding our custom css definitions, application css would be used only for definitions like imports.

 @import 'bootstrap-sprockets';
-@import 'bootstrap'; 
+@import 'bootstrap';